Source/Bugsnag.h

```diff
@@ -264,6 +266,16 @@ static NSString *_Nonnull const BugsnagSeverityInfo = @"info";
  */
 + (BOOL)resumeSession;
 
+/**
+* Return the metadata for a specific named section
+*
+* @param section The name of the section
+* @returns The mutable dictionary representing the metaadata section, if it
+*          exists, or nil if not.
+*/
++ (NSMutableDictionary *_Nullable)getMetadata:(NSString *_Nonnull)section
+    NS_SWIFT_NAME(getMetadata(_:));
+
```

Comment:
The `Bugsnag` interface is missing a couple of the method variants that are defined in the notifier spec. For instance, 

- `getMetadata:section:key` could return an object of type `id`
- `clearMetadata:section:key` could clear a specific value
- `addMetadata:section` could take an `NSDictionary`, acting as syntactic sugar for `addMetadata:section:key`

Reply:
The task only calls for the single method to be implemented.  Required lower-level methods (in BugsnagMetadata for instance) have been implemented in separate branches as part of other tasks and it makes more sense to implement the missing methods once those have been merged.  I agree with you though, and think that we should realign the existing tasks to the spec, one class/section per task.

Reply:
Happy with this approach as long as the task is tracked somewhere.